common/djangoapps/util/tests/test_submit_feedback.py

```diff
@@ -520,6 +520,48 @@ def test_valid_request_auth_user_with_enterprise_info(self, course_id, enrolled,
         self._assert_zendesk_called(zendesk_mock_instance, ticket_id, ticket, ticket_update)
         self._assert_datadog_called(datadog_mock, datadog_tags)
 
+    @httpretty.activate
+    @override_settings(ZENDESK_CUSTOM_FIELDS=TEST_ZENDESK_CUSTOM_FIELD_CONFIG, ENABLE_ENTERPRISE_INTEGRATION=True)
+    def test_request_with_anonymous_user_without_enterprise_info(self, zendesk_mock_class, datadog_mock):
+        """
+        Test tags related to enterprise should not be there in case an unauthenticated user.
+        """
+        ticket_id = 42
+        self.mock_enterprise_learner_api()
+        user = self._anon_user
```

Comment:
@zubair-arbi it is the pre-exist name in that test file, changes this to `_anonymous` will effect many other tests as well.